masterlist.yaml

```
@@ -34013,3 +34020,28 @@ plugins:
         util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
         itm: 4
         udr: 40
+
+  - name: 'ImpeREAL Empire Unique Cities - Sutch.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/37867/'
+        name: 'ImpeREAL Empire Unique Cities - Sutch'
+    dirty:
+      - <<: *quickClean
+        crc: 0xEB59AB31
+        util: '[TES4Edit v4.0.4](https://www.nexusmods.com/oblivion/mods/11536)'
+        itm: 2
+    clean:
+      - crc: 0x0872D0D1
+        util: 'TES4Edit v4.0.4'
+
+  - name: 'Sutch Coblized.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/52617'
```

```suggestion
      - link: 'https://www.nexusmods.com/oblivion/mods/52617/'
```

URLs should have trailing slashes if possible for consistency & convenience (it makes it easier to search for Nexus IDs).